### ISSUE TITLE
Apache config not working, or perhaps invenio itself not working (seeking advice)

### ISSUE THREAD

**Issue by solarchemist** [materialscloud-org/big-map-archive#36]
Running an LXD virtual machine (called `callisto`, running Ubuntu 20.04, kernel 5.15, Python 3.8.10, Docker 20.10, docker-compose 1.25, Node 14.21 and Apache 2.4.41) on a bare-metal host running Ubuntu 18.04 (x64) with Apache 2.4.29 and LXD 5.11 (snap). I chose 20.04 on the VM to suit this project because it runs Python 3.8 by default, which is required by this project.

On the LXD VM I installed Invenio RDM by cloning [this repo (latest commit)](https://github.com/materialscloud-org/big-map-archive/commit/9521694e530527935aec1e15958c7b098c7611a3), docker compose up and installing the invenio-cli (I also uninstalled the apt package virtualenv because it clashed with the same pip package installed when installing invenio-cli), like so:
```
solarchemist@callisto:~
$ git clone https://github.com/materialscloud-org/big-map-archive.git inveniordm
$ cd inveniordm
$ docker-compose up --detach
$ python3.8 -m pip install --user invenio-cli
$ sudo apt remove python3-virtualenv
```

Following the [InvenioRDM installation steps](https://inveniordm.docs.cern.ch/install/), I then ran `invenio-cli install` which 
failed with this error:
```
solarchemist@callisto:~/inveniordm
$ invenio-cli install
Installing python dependencies... Please be patient, this operation might take some time...
Creating a virtualenv for this project...
Pipfile: /home/solarchemist/inveniordm/Pipfile
Using /usr/bin/python3.8 (3.8.10) to create virtualenv...
⠼ Creating virtual environment...created virtual environment CPython3.8.10.final.0-64 in 864ms
  creator CPython3Posix(dest=/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/solarchemist/.local/share/virtualenv)
    added seed packages: pip==23.1.2, setuptools==67.7.2, wheel==0.40.0
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator

✔ Successfully created virtual environment!
Virtualenv location: /home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV
Installing dependencies from Pipfile.lock (47f202)...
Installing dependencies from Pipfile.lock (47f202)...
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
All dependencies are now up-to-date!
Updating instance path...
Failed to install dependencies.
Errors: Traceback (most recent call last):
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3108, in _dep_map
    return self.__dep_map
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2901, in __getattr__
    raise AttributeError(attr)
AttributeError: _DistInfoDistribution__dep_map

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 35, in __init__
    parsed = parse_requirement(requirement_string)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 64, in parse_requirement
    return _parse_requirement(Tokenizer(source, rules=DEFAULT_RULES))
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 82, in _parse_requirement
    url, specifier, marker = _parse_requirement_details(tokenizer)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 120, in _parse_requirement_details
    specifier = _parse_specifier(tokenizer)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_parser.py", line 209, in _parse_specifier
    tokenizer.consume("WS")
  File "/usr/lib/python3.8/contextlib.py", line 120, in __exit__
    next(self.gen)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_tokenizer.py", line 183, in enclosing_tokens
    self.raise_syntax_error(
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/_tokenizer.py", line 163, in raise_syntax_error
    raise ParserSyntaxError(
pkg_resources.extern.packaging._tokenizer.ParserSyntaxError: Expected closing RIGHT_PARENTHESIS
    tinycss2 (>=1.1.0<1.2) ; extra == 'css'
             ~~~~~~~~^

The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/bin/invenio", line 8, in <module>
    sys.exit(cli())
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/cli.py", line 579, in main
    return super().main(*args, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/cli.py", line 426, in decorator
    with __ctx.ensure_object(ScriptInfo).load_app().app_context():
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/cli.py", line 382, in load_app
    app = self.create_app()
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_base/app.py", line 163, in create_cli_app
    app = create_app(debug=get_debug_flag())
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_base/app.py", line 99, in _create_app
    config_loader(app, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_app/factory.py", line 60, in config_loader
    invenio_config_loader(app, **kwargs_config)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_config/utils.py", line 54, in _config_loader
    InvenioConfigEntryPointModule(app=app)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_config/entrypoint.py", line 35, in __init__
    self.init_app(app)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_config/entrypoint.py", line 45, in init_app
    app.config.from_object(ep.load())
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2516, in load
    self.require(*args, **kwargs)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2539, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 832, in resolve
    new_requirements = dist.requires(req.extras)[::-1]
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2821, in requires
    dm = self._dep_map
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3110, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3120, in _compute_dependencies
    reqs.extend(parse_requirements(req))
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3173, in __init__
    super(Requirement, self).__init__(requirement_string)
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 37, in __init__
    raise InvalidRequirement(str(e)) from e
pkg_resources.extern.packaging.requirements.InvalidRequirement: Expected closing RIGHT_PARENTHESIS
    tinycss2 (>=1.1.0<1.2) ; extra == 'css'
             ~~~~~~~~^
```

So, that was unexpected. And I did check requirements by the way, returned OK:
```
solarchemist@callisto:~/inveniordm
$ invenio-cli check-requirements --development
Checking pre-requirements...
Checking Python version...
Python version OK. Got 3.8.10.
Checking Pipenv is installed...
Pipenv OK. Got version 2023.5.19.
Checking Docker version...
Docker version OK. Got 20.10.21.
Checking Docker Compose version...
Docker Compose version OK. Got 1.25.0.
Checking Node version...
Node version OK. Got 14.21.3.
Checking NPM version...
NPM version OK. Got 6.14.18.
Checking ImageMagick version...
ImageMagick version OK. Got 6.9.10.
Checking git version...
git version OK. Got 2.25.1.
All requisites are fulfilled.
```

Searching on the web for the error messages above gave me no clues to work with, but [this slightly related post in the Invenio forum](https://invenio-talk.web.cern.ch/t/installation-issues/224) suggested to wipe the `Pipfile.lock`. Wiped and reprovisioned the VM (starting afresh), and after deleting the lockfile `invenio-cli install` threw what looked like a bunch of errors, but still reported success? (please see [attached invenio-cli-install.log, around 2400 lines](https://github.com/materialscloud-org/big-map-archive/files/11581928/invenio-cli-install.log)).

Thinking that proof is in the eating of the pudding, I went ahead and attempted to run invenio:
```
solarchemist@callisto:~/inveniordm
$ invenio-cli run
redis up and running!
postgresql up and running!
search up and running!
Containers started and healthy.
Starting celery worker...
Starting up local (development) server...
Instance running!
Visit https://127.0.0.1:5000
Traceback (most recent call last):
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/bin/celery", line 8, in <module>
    sys.exit(main())
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/celery/__main__.py", line 14, in main
    from celery.bin.celery import main as _main
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/celery/bin/celery.py", line 34, in <module>
    from celery.bin.worker import worker
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/celery/bin/worker.py", line 14, in <module>
    from celery.concurrency.base import BasePool
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/celery/concurrency/base.py", line 12, in <module>
    from celery.utils import timer2
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/celery/utils/timer2.py", line 14, in <module>
    from kombu.asynchronous.timer import Entry
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/kombu/asynchronous/__init__.py", line 7, in <module>
    from .hub import Hub, get_event_loop, set_event_loop
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/kombu/asynchronous/hub.py", line 19, in <module>
    from .timer import Timer
  File "/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/kombu/asynchronous/timer.py", line 22, in <module>
    from backports.zoneinfo import ZoneInfo
ModuleNotFoundError: No module named 'backports'
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-05-27 13:41:14,278] DEBUG in entrypoint: Loading config for entry point invenio_app_rdm = invenio_app_rdm.config
[2023-05-27 13:41:15,202] DEBUG in ext: Flask-DebugToolbar extension not installed.
/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/scaffold.py:50: DeprecationWarning: 'before_app_first_request' is deprecated and will be removed in Flask 2.3. Use 'record_once' instead to run setup code when registering the blueprint.
  return f(self, *args, **kwargs)

'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-05-27 13:41:15,631] DEBUG in entrypoint: Loading config for entry point invenio_app_rdm = invenio_app_rdm.config
/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_accounts/ext.py:244: UserWarning: The setup method 'route' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  blueprint.route("/security/", methods=["GET"])(security)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/scaffold.py:449: UserWarning: The setup method 'add_url_rule' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  self.add_url_rule(rule, endpoint, f, **options)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/blueprints.py:493: UserWarning: The setup method 'record' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  self.record(

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_accounts/ext.py:245: UserWarning: The setup method 'route' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  blueprint.route("/sessions/revoke/", methods=["POST"])(revoke_session)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/scaffold.py:50: DeprecationWarning: 'before_first_request' is deprecated and will be removed in Flask 2.3. Run setup code while creating the application instead.
  return f(self, *args, **kwargs)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:43: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  class JSONEncoder(app.json_encoder):

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:129: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  app.json_encoder = get_lazystring_encoder(app)

[2023-05-27 13:41:15,750] DEBUG in ext: Flask-DebugToolbar extension not installed.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on https://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-05-27 13:41:17,813] DEBUG in entrypoint: Loading config for entry point invenio_app_rdm = invenio_app_rdm.config
[2023-05-27 13:41:18,824] DEBUG in ext: Flask-DebugToolbar extension not installed.
/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask_babelex/__init__.py:606: DeprecationWarning: '_request_ctx_stack' is deprecated and will be removed in Flask 2.3. Use 'g' to store data, or 'request_ctx' to access the current context.
  ctx = _request_ctx_stack.top

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask_babelex/__init__.py:506: DeprecationWarning: '_request_ctx_stack' is deprecated and will be removed in Flask 2.3. Use 'g' to store data, or 'request_ctx' to access the current context.
  ctx = _request_ctx_stack.top

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:43: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  class JSONEncoder(app.json_encoder):

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:129: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  app.json_encoder = get_lazystring_encoder(app)

'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-05-27 13:41:19,157] DEBUG in entrypoint: Loading config for entry point invenio_app_rdm = invenio_app_rdm.config
/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_accounts/ext.py:244: UserWarning: The setup method 'route' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  blueprint.route("/security/", methods=["GET"])(security)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/scaffold.py:449: UserWarning: The setup method 'add_url_rule' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  self.add_url_rule(rule, endpoint, f, **options)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/blueprints.py:493: UserWarning: The setup method 'record' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  self.record(

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_accounts/ext.py:245: UserWarning: The setup method 'route' can no longer be called on the blueprint 'invenio_accounts'. It has already been registered at least once, any changes will not be applied consistently.
Make sure all imports, decorators, functions, etc. needed to set up the blueprint are done before registering it.
This warning will become an exception in Flask 2.3.
  blueprint.route("/sessions/revoke/", methods=["POST"])(revoke_session)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/flask/scaffold.py:50: DeprecationWarning: 'before_first_request' is deprecated and will be removed in Flask 2.3. Run setup code while creating the application instead.
  return f(self, *args, **kwargs)

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:43: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  class JSONEncoder(app.json_encoder):

/home/solarchemist/.local/share/virtualenvs/inveniordm-bvtLDiTV/lib/python3.8/site-packages/invenio_i18n/ext.py:129: DeprecationWarning: 'app.json_encoder' is deprecated and will be removed in Flask 2.3. Customize 'app.json_provider_class' or 'app.json' instead.
  app.json_encoder = get_lazystring_encoder(app)

[2023-05-27 13:41:19,519] DEBUG in ext: Flask-DebugToolbar extension not installed.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
 * Debugger is active!
 * Debugger PIN: 179-560-413
```
which says **instance running** and `Running on https://127.0.0.1:5000` but also produces lots of warnings and what looks like errors too. Port 5000 is now indeed occupied by a Python process, but so far all my attempts to access the web app have failed, but I can't determine whether that's because my Apache virtualhost config files are bad or whether the invenio-rdm job is in fact broken.
On the bare-metal host I have the following enabled Apache vhost (this server acts as reverse proxy and TLS termination point):
```
$ cat /etc/apache2/sites-enabled/invenio.example.se.conf 
<VirtualHost *:80>
   ServerAdmin solarchemist@example.se
   ServerName invenio.example.se
   Redirect permanent / https://invenio.example.se
   LogLevel info
   CustomLog ${APACHE_LOG_DIR}/invenio.example.se_access.log combined
   ErrorLog ${APACHE_LOG_DIR}/invenio.example.se_error.log
</VirtualHost>

<VirtualHost *:443>
   ServerAdmin solarchemist@example.se
   ServerName invenio.example.se
   ProxyPreserveHost On
   ProxyRequests Off
   ProxyPass / http://10.238.116.2:80/
   ProxyPassReverse / http://10.238.116.2:80/
 
   SSLEngine on
   SSLCertificateKeyFile /etc/letsencrypt/live/invenio.example.se/privkey.pem
   SSLCertificateFile    /etc/letsencrypt/live/invenio.example.se/cert.pem
   SSLCACertificateFile  /etc/letsencrypt/live/invenio.example.se/chain.pem
   LogLevel info
   CustomLog ${APACHE_LOG_DIR}/invenio.example.se_access.log combined
   ErrorLog ${APACHE_LOG_DIR}/invenio.example.se_error.log
</VirtualHost>
```
and on the LXD VM I created the following Apache vhost:
```
$ cat /etc/apache2/sites-enabled/invenio.example.se.conf 
<VirtualHost *:80>
   ServerAdmin solarchemist@example.se
   ServerName invenio.example.se
   DocumentRoot /home/solarchemist/inveniordm
   <Directory /home/solarchemist/inveniordm/static>
      Options FollowSymLinks MultiViews
      AllowOverride None
      Require all granted
   </Directory>
   ProxyPreserveHost On
   ProxyRequests Off
   ProxyPass / http://127.0.0.1:5000/
   ProxyPassReverse / http://127.0.0.1:5000/
   LogLevel warn
   LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\" %D" combined_with_timing
   CustomLog ${APACHE_LOG_DIR}/invenio.example.se_access.log combined_with_timing
   ErrorLog ${APACHE_LOG_DIR}/invenio.example.se_error.log
</VirtualHost>
```

In the web browser I get "Proxy Error, reason: error reading from remote server" and in the Apache error log I get (while `invenio-cli run` is left running, of course):
```
[Sat May 27 14:49:47.396750 2023] [proxy_http:error] [pid 184676:tid 140604520515328] (104)Connection reset by peer: [client 10.238.116.1:52148] AH01102: error reading status line from remote server 127.0.0.1:5000
[Sat May 27 14:49:47.397318 2023] [proxy:error] [pid 184676:tid 140604520515328] [client 10.238.116.1:52148] AH00898: Error reading from remote server returned by /
[Sat May 27 14:49:47.677650 2023] [proxy_http:error] [pid 184676:tid 140604512122624] (104)Connection reset by peer: [client 10.238.116.1:52148] AH01102: error reading status line from remote server 127.0.0.1:5000, referer: https://invenio.example.se/
[Sat May 27 14:49:47.736211 2023] [proxy:error] [pid 184675:tid 140604520515328] [client 10.238.116.1:52160] AH00898: Error reading from remote server returned by /favicon.ico, referer: https://invenio.example.se/
```
and the Apache error log on the proxy server:
```
[Sat May 27 14:49:47.371359 2023] [ssl:info] [pid 11914:tid 140507707537152] [client 128.228.198.180:45602] AH01964: Connection to child 16 established (server invenio.example.se:443)
[Sat May 27 14:49:47.371415 2023] [ssl:info] [pid 11915:tid 140506424141568] [client 128.228.198.180:45594] AH01964: Connection to child 86 established (server invenio.example.se:443)
[Sat May 27 14:49:48.054968 2023] [proxy_http:error] [pid 11915:tid 140506432534272] (20014)Internal error (specific information not available): [client 128.228.198.180:45594] AH01102: error reading status line from remote server 10.238.116.2:80, referer: https://invenio.example.se/
```

In summary, I would love to hear from other admins: are all these errors during `install` and `run` normal? Why did `install` not work with the project-supplied `Pipfile.lock`? Should I use different versions of any dependencies? Or is the problem with my Apache vhost config files?

Any and all pointers greatly appreciated.
